### PR TITLE
Fix delegate num

### PR DESCRIPTION
**Pull request by chen-eugene**

### CODE REVIEW

**Comment by chen-eugene** [comment]
Only show the total number of delegate [#1764](https://github.com/iotexproject/iotex-explorer/issues/1764)
image

File: src/shared/home/stats/candidates-card.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ export const CandidatesCard = (props: QueryResult): JSX.Element => {
       productivityBase: number;
     }>;
   } = data || {};
-  const consensusDelegateCount = 36;
-  const candidatesCount = bpCandidates.length;
+  const consensusDelegateCount = bpCandidates.length;
+  // const candidatesCount = bpCandidates.length;
```
Comment (guo) [inline src/shared/home/stats/candidates-card.tsx:19]: should keep this.

File: src/shared/home/stats/candidates-card.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const CandidatesCard = (props: QueryResult): JSX.Element => {
       value={consensusDelegateCount}
```
Comment (guo) [inline src/shared/home/stats/candidates-card.tsx:28]: replace this one.

File: src/shared/home/stats/candidates-card.tsx (hunk 1 above)
Comment (guo) [reply src/shared/home/stats/candidates-card.tsx:19]: the length is the all delegateCount.

File: src/shared/home/stats/candidates-card.tsx (hunk 2 above)
Comment (chen-eugene) [reply src/shared/home/stats/candidates-card.tsx]: has been replaced by candidatesCount

File: src/shared/home/stats/candidates-card.tsx (hunk 1 above)
Comment (chen-eugene) [reply src/shared/home/stats/candidates-card.tsx]: delete consensusDelegateCount

**Review by anshuomomo — APPROVED** [review]

**Pull request merged**